get.sh

```diff
@@ -162,10 +162,14 @@ getPackage() {
                 rm "$targetFile"
             fi
 
-            if [ -n "$ALIAS_NAME" ]; then
-                if [ ! -L $BINLOCATION/$ALIAS_NAME ]; then
-                    ln -s $BINLOCATION/$REPO $BINLOCATION/$ALIAS_NAME
-                    echo "Creating alias '$ALIAS_NAME' for '$REPO'."
+            if [ $(which $ALIAS_NAME) ]; then
+                echo "There is already a command '$ALIAS_NAME' in the path, do NOT create alias"
```

Comment:
Could you also change the message, just so that it's consistent with the other one ?
```suggestion
                echo "There is already a command '$ALIAS_NAME' in the path, NOT creating alias"
```